### PR TITLE
Add GitHub actions

### PR DESCRIPTION
**Pull request by AmoabaKelvin**

### CODE REVIEW

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+      - name: ⎔ Setup node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
+
+      - name: Set up pnpm
+        uses: pnpm/action-setup@v2
+        with:
+          version: 7
```
Comment (thibaultleouay) [inline .github/workflows/lint.yml]: We are using pnpm 8

File: .github/workflows/lint.yml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    name: ⬣ ESLint
+    runs-on: ubuntu-latest
+    steps:
+      - name: ⬇️ Checkout repo
+        uses: actions/checkout@v2
+
+      - name: ⎔ Setup node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
```
Comment (thibaultleouay) [inline .github/workflows/lint.yml]: We are using node 18

File: package.json [hunk 3]
```diff
@@ -4,7 +4,9 @@
     "build": "turbo run build",
     "dev": "turbo run dev",
     "lint": "turbo run lint",
+    "lint:strict": "eslint --max-warnings=0 apps/**/*.{ts,tsx}",
```
Comment (thibaultleouay) [inline package.json:7]: Can we take advantage of the turbo repo ? And lint every package?

File: package.json [hunk 4]
```diff
@@ -4,7 +4,9 @@
...
     "format": "prettier --write \"**/*.{ts,tsx,md}\"",
+    "format:check": "prettier -c \"**/*.{ts,tsx,md}\"",
```
Comment (thibaultleouay) [inline package.json:9]: Same for prettier we can run it in every package and run it with turbo ?

**Review by thibaultleouay — COMMENTED** [review]
Thanks @AmoabaKelvin for your pr 🔥

I'd love it if we could take advantage of the turbo repository

File: package.json (hunk 3 above)
Comment (AmoabaKelvin) [reply package.json:7]: Sure

File: package.json (hunk 4 above)
Comment (AmoabaKelvin) [reply package.json:9]: For prettier, I am currently not sure we can run that with turbo, I went through several articles and what they had implemented was equivalent to what I wrote.

Comment (AmoabaKelvin) [reply package.json:9]: @thibaultleouay please provide me some updates so I proceed. Thanks

**Comment by thibaultleouay** [comment]
I have made some update and I will merge it 
Thanks again @AmoabaKelvin

**Pull request merged**